cli/version_0.4.0/src/main/scala/scala/scalanative/cli/utils/VersionSpecificOptionsIncluder.scala

```diff
@@ -0,0 +1,14 @@
+package scala.scalanative.cli.utils
+
+import scala.scalanative.build.NativeConfig
+
+import scala.scalanative.cli.options.CliOptions
+
+object VersionSpecificOptionsIncluder {
```

Comment:
I think we should limit the scope of those calls and make it `private[utils]` (also in other files)

Reply:
Or even better make the whole object package private